models/repo/avatar.go

```diff
@@ -24,6 +24,11 @@ func (repo *Repository) CustomAvatarRelativePath() string {
 	return repo.Avatar
 }
 
+// ExistRepoWithAvatar returns true if there is a user with this Avatar
+func ExistRepoWithAvatar(ctx context.Context, avatar string) (bool, error) {
```

Comment:
We should probably clarify that this is a filepath.
When looking through the code, this wasn't immediately clear for me.